tools/setup_launcher_py3.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
Quick question: why the rather complex installation path? I feel like i am missing something; can't we just use the normal `setup.py install` procedure, and if the user wants a custom directory, they will set up a virtualenv?

Reply:
I tried that and it kept breaking on my macOS default install :-(